### PR TITLE
GUACAMOLE-436: Update website with respect to graduation.

### PR DESCRIPTION
**Pull request by mike-jumper**
This change:

* Removes "incubating" and the corresponding disclaimer
* Updates things to refer to the PMC, rather than the PPMC and IPMC
* Updates the domains of the mailing lists to `guacamole.apache.org`
* Notes that "Apache Guacamole", "Guacamole", and the Guacamole logo are trademarks of the ASF (albeit unregistered)

### CODE REVIEW

**Comment by jmuehlner** [comment]
Are we going to change the name of this repository at some point?

**Comment by mike-jumper** [comment]
They will change, yes, but we can't do that ourselves directly. It is one of the tasks that Infra must take care of for new TLPs. In our case:

https://issues.apache.org/jira/browse/INFRA-15505

Once that happens, then we can update the various links pointing to the repos.

File: open-source.md [hunk 1]
```diff
@@ -104,19 +103,19 @@ a committer not otherwise directly involved in those changes.
...
+PMC members
+-----------
 
-The Apache Guacamole PPMC is responsible for project oversight. PPMC members
-participate in discussions on the [mailing lists](/support/#mailing-lists) and
-vote on release candidates, new committers, and new PPMC members. Like
-[committers](#committers) (who are implicitly PPMC members), the PPMC is
-responsible for identifying members of the community who have shown
-[merit](#meritocracy) warranting committership or PPMC membership.
+The Apache Guacamole PMC is made up of committers and is responsible for
```
Comment (jmuehlner) [inline open-source.md]: This line here makes it seem like the committers are the same set as the PMC members, but a further sentence below states: "The PMC is responsible for identifying members ... warranting committership and/or
PMC membership." So are they not equivalent then? Can somebody be a PMC member but not a committer? This is a bit unclear here.

**Comment by necouchman** [comment]
With the exception of @jmuehlner comment above, this looks good to me.  I don't see anything else that needs tweaking.

File: open-source.md (hunk 1 above)
Comment (mike-jumper) [reply open-source.md]: > So are they not equivalent then? Can somebody be a PMC member but not a committer?

Nope. All PMC members are by definition committers, but not all committers are PMC members. From https://www.apache.org/foundation/how-it-works.html#roles:

>
> COMMITTER
> ---------
> 
> A *committer* is a developer that was given write access to the code repository and has a signed [Contributor License Agreement (CLA)](http://www.apache.org/licenses/#clas) on file. They have an apache.org mail address. Not needing to depend on other people for the patches, they are actually making short-term decisions for the project. The PMC can (even tacitly) agree and approve it into permanency, or they can reject it. Remember that the PMC makes the decisions, not the individual committers.
>
> PMC MEMBER
> ----------
> A *PMC member* is a developer or a committer that was elected due to merit for the evolution of the project and demonstration of commitment. They have write access to the code repository, an apache.org mail address, the right to vote for the community-related decisions and the right to propose an active user for committership. The PMC as a whole is the entity that controls the project, nobody else. In particular, the PMC must vote on any formal release of their project's software products.
>

The distinction between PMC member and committer is defined at the foundation level. Though PPMC/committer equivalency was a simple decision in our podling days, PMC membership for any project requires board approval (or at least ... 72-hour lack of board disapproval: http://www.apache.org/dev/pmc.html#newpmc).

Comment (jmuehlner) [reply open-source.md]: Ok, so this first sentence should be updated to reflect the fact that not all PMC members are committers then, yeah?

Comment (mike-jumper) [reply open-source.md]: Sure. I'll try to reword to clarify.

Comment (mike-jumper) [reply open-source.md]: Wait ... sorry. I misread.

> ... to reflect the fact that not all PMC members are committers ...

No. All PMC members are committers. Not all committers are PMC members.

Comment (jmuehlner) [reply open-source.md]: Ah ok, right. So I guess this looks ok afterall!

Comment (mike-jumper) [reply open-source.md]: 🎉

**Comment by jmuehlner** [comment]
No more concerns! Already merged. :)

**Comment by necouchman** [comment]
> No more concerns! Already merged. :)

Cool!
:+1:

**Pull request merged**